WORKSPACE

```diff
@@ -113,9 +120,16 @@ new_git_repository(
     remote = "https://github.com/googleapis/googleapis.git",
 )
 
+# Dependency for org_golang_google_grpc
+new_go_repository(
```

Comment:
Ok. Sounds reasonable. Thanks. I bet @ZackButcher is looking forward to merging his WORKSPACE changes in with this ;).